app/models/security.rb

```diff
@@ -2,6 +2,7 @@ class Security < ApplicationRecord
   before_save :upcase_ticker
 
   has_many :trades, dependent: :nullify, class_name: "Account::Trade"
+  has_many :prices, dependent: :destroy, class_name: "Security::Price"
```

Comment:
Namespacing _should_ handle this to where we don't need a `class_name` for `prices`